util/worker.go

```
@@ -0,0 +1,141 @@
+		errorsSlice: make([]error, 0),
+	}
+}
+
+// Start initializes the worker pool
+func (wp *WorkerPool) Start() {
+	wp.mu.Lock()
+	if wp.isRunning {
+		wp.mu.Unlock()
+		return
+	}
+	wp.isRunning = true
```

golangci-lint (1.62.2)

assignments should only be cuddled with other assignments

(wsl)